ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -141,9 +141,9 @@ private Router setupRouter() {
 
     PortedEndpoints.setupEndpoints(endpoints, server, router);
 
-    if (proxyHandler != null) {
-      proxyHandler.setupRoutes(router);
-    }
+    // if (proxyHandler != null) {
```

Comment:
This does not look like it's meant to be commented out...

Reply:
That's removed in another PR/commit (?). At least I don't see this after rebasing.